README.md

```diff
@@ -1 +1,59 @@
-# ToDoList
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# To-Do List
+> This web application provides users a convenient way to keep track of their daily routine works. Users can add and remove tasks from the list of tasks they curate on the website.
+
+
+![SCREENSHOT](./assets/todo.GIF)
+
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+
+
+## Clone and review it
+
+Run the following commands to clone and run it.
+
+### To clone the repository
+
+  `git@github.com:HAMMAS-SALEEM/ToDoList.git`
+
+### To enter Directory
+
+`cd ToDoList`
+
+### To Install Packages
+
+`npm install`
+
+### To run 
```

Comment:
- [ ] [Required] I think you meant `To open with vs-code` here. `code .` opens the project on visual studio code application I guess. What's your thought on this, please?